CODEOWNERS

```diff
@@ -3,4 +3,7 @@
 # the last matching pattern has the most precendence.
 
 # Core team members from IBM
-* @kjdelisle @jannyHou @loay @b-admike @ssh24 @virkt25 @dhmlau @zbarbuto
+* @jannyHou @b-admike @virkt25 @dhmlau @zbarbuto
+
+# Alumni members
+* @kjdelisle @loay @ssh24
```

Comment:
I think this line needs to be commented out so they aren't treated as active members by GitHub.

Reply:
you're right. otherwise the last line will take precedence.  thanks!